wgsl/index.bs

```diff
@@ -677,36 +677,37 @@ type.
 </pre>
 * type must be `f32`, `i32` or `u32`
 
-### Read-only Storage Texture Types ### {#texture-ro}
+### Storage Texture Types ### {#texture-storage}
 
 A <dfn noexport>read-only texture</dfn> is a texture from which individual texels
-may be read, without the use of a sampler.
-See [[#texture-builtin-functions]].
+may be read, without the use of a sampler. A <dfn noexport>write-only texture</dfn>
+is a texture to which individual texels may be written. See [[#texture-builtin-functions]].
 
 <pre class='def'>
-`texture_ro_1d<image_storage_type>`
+`texture_storage_1d<read|write, image_storage_type>`
   %1 = OpTypeImage type(image_storage_type) 1D 0 0 0 2 image_storage_type
 
-`texture_ro_1d_array<image_storage_type>`
+`texture_storage_1d_array<read|write, image_storage_type>`
   %1 = OpTypeImage type(image_storage_type) 1D 0 1 0 2 image_storage_type
 
-`texture_ro_2d<image_storage_type>`
+`texture_storage_2d<read|write, image_storage_type>`
   %1 = OpTypeImage type(image_storage_type) 0 0 0 2 image_storage_type
 
-`texture_ro_2d_array<image_storage_type>`
+`texture_storage_2d_array<read|write, image_storage_type>`
   %1 = OpTypeImage type(image_storage_type) 0 1 0 2 image_storage_type
 
-`texture_ro_3d<image_storage_type>`
+`texture_storage_3d<read|write, image_storage_type>`
   %1 = OpTypeImage type(image_storage_type) 3D 0 0 0 2 image_storage_type
 </pre>
-* type(image_storage_type) is one of `f32`, `i32`, or `u32`, depending on the storage type.
+* type(image_storage_type) for a `read` texture is one of `f32`, `i32`, or `u32`, depending on the storage type.
+* type(image_storage_type) for a `write` texture is one of `f32`, `i32`, or `u32`, depending on the storage type or `void`.
 
-When mapping to SPIR-V, a read-only storage texture variable must also have a `NonWritable` decoration.
-For example:
+When mapping to SPIR-V, a read-only storage texture variable must also have a
+`NonWritable` decoration. For example:
 
-<div class='example' heading='Mapping a texture_ro_1d variable to SPIR-V'>
+<div class='example' heading='Mapping a texture_storage_1d read variable to SPIR-V'>
```

Comment:
The heading caption has weird phrasing.
Suggest

> Mapping a readable texture_storage_1d variable to SPIR-V

Reply:
done